spec/student_spec.rb

```diff
@@ -20,5 +20,13 @@
     it 'Creates a new student with a classroom' do
       expect(student.classroom).to eq('Math')
     end
+
+    describe '#play_hooky' do
+  it 'should show play_hooky symbol' do
+    student = Student.new(classroom: 6, age: 13, parent_permission: 'Yes', name: 'Mark')
+    expect(student.play_hooky).to eq '¯\(ツ)/¯'
+  end
+end
   end
+
```

Comment:
[REQUIRED] Hello team, great job so far but as suggested by the previous review it seems some methods have not been tested.
Let's test, `classroom=(classroom) ` methods for this class as stated in the project requirements.